builtin/logical/pki/storage.go

```diff
@@ -0,0 +1,104 @@
+package pki
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/vault/sdk/helper/certutil"
+	"github.com/hashicorp/vault/sdk/helper/errutil"
+	"github.com/hashicorp/vault/sdk/logical"
+)
+
+const (
+	pkiKeyPrefix    = "/config/key/"
+	pkiIssuerPrefix = "/config/issuer/"
+)
+
+var (
+	emptyPkiKey = pkiKey{}
+	emptyIssuer = pkiIssuer{}
+)
+
+type pkiKeyId string
```

Comment:
The argument I've generally seen for eliding the `pki` name from the variables is that you're already in the `pki` package, so that's implied. Would there be conflicts if we drop the pki prefix everywhere? Just curious :-)

Reply:
Yeah I was really torn, I could definitely drop the pki references a bit everywhere within this package. Will do.